CMakeLists.txt

```diff
@@ -1,77 +1,95 @@
-# portmidi
-# Roger B. Dannenberg
-# 20 Sep 2009
+cmake_minimum_required(VERSION 3.0)
 
-cmake_minimum_required(VERSION 2.6)
+project(PortMIDI VERSION 1.0)
```

Comment:
I'm not sure how to handle this. Previously PortMIDI used SVN revision numbers for versioning.

Reply:
Shouldn't be a problem, since it's not really used anywhere.

Reply:
Yeah, the CMake project version isn't used anywhere. But I want to make this the same as whatever version it will be tagged as..

Reply:
The last SVN revision was 234, so maybe this should be 235?

Reply:
If we add a tag to the SVN Revision 0, we can continue the counting using git describe.
Alternative we can treat version 235 as tag and start with 235.1 or such.

Reply:
I think I'll start the versions as 235 to be safe. I don't want to setup users to not get an updated package because of a changed version scheme.